swim/member.go

```diff
@@ -107,6 +110,8 @@ func statePrecedence(s string) int {
 		return 2
 	case Leave:
 		return 3
+	case Tombstone:
```

Comment:
Oh man, are we still doing this? Can we make these states proper integers so we can just do `Tomestone > Leave` etc?